src/coreclr/tools/superpmi/superpmi-shim-collector/icorjitinfo.cpp

```diff
@@ -123,9 +129,12 @@ void interceptor_ICJI::reportInliningDecision(CORINFO_METHOD_HANDLE inlinerHnd,
                                               CorInfoInline         inlineResult,
                                               const char*           reason)
 {
-    mc->cr->AddCall("reportInliningDecision");
-    original_ICorJitInfo->reportInliningDecision(inlinerHnd, inlineeHnd, inlineResult, reason);
-    mc->cr->recReportInliningDecision(inlinerHnd, inlineeHnd, inlineResult, reason);
+    if (inlineResult == INLINE_PASS || inlineResult == INLINE_FAIL|| inlineResult == INLINE_NEVER)
+    {
```

Comment:
Why not just drop this condition and always do the pass-through/recReportInliningDecision call? I don't think it would hurt.

Reply:
Yep, seems to work.